pom.xml

```diff
@@ -363,6 +384,9 @@
                             </execution>
                         </executions>
                         <configuration>
+                            <!-- Make javadoc use the de-lomboked code -->
+                            <sourcepath>target/generated-sources/delombok</sourcepath>
```

Comment:
Is this a standard directory for the lombok plugin?

Reply:
Yup. See here: http://anthonywhitford.com/lombok.maven/lombok-maven-plugin/delombok-mojo.html#outputDirectory